src/logging.cpp

```diff
@@ -29,45 +34,176 @@ bl::sources::severity_logger<int> warning(3);  // Strange events
 bl::sources::severity_logger<int> error(4);  // Recoverable errors
 bl::sources::severity_logger<int> fatal(5);  // Unrecoverable errors
 
-/**
- * @brief Flush the log.
- *
- * EXAMPLES:
- * ```cpp
- * log_flush();
- * ```
- */
-void
-log_flush() {
-  sink->flush();
-}
+struct NoDelete {
```

Comment:
This can be replaced with `boost::empty_deleter` from `#include <boost/utility/empty_deleter.hpp>`

Reply:
Do you mean `boost::null_deleter()` from `#include <boost/core/null_deleter.hpp>`?

CLion complains it cant find `<boost/utility/empty_deleter.hpp>`

Reply:
Yup, probably was updated in latest versions